App/Course App/Course App/Views/UIKit/UICollectionViewCell/HorizontalCollectionSwiftUICell.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  HorizontalCollectionSwiftUICell.swift
+//  Course App
+//
+//  Created by Work on 22.05.2024.
+//
+
+import Foundation
+import SwiftUI
+import UIKit
+
+// My testing purposes
+class HorizontalCollectionSwiftUICell: UICollectionViewCell {
```

Comment:
This class seems to be unused for now. Also no need to use HostingController, there is hosting configuration available for cells usings SwiftUI

Reply:
now is removed